app/src/androidTest/java/kk/ljq/hhh/retrofit2/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package kk.ljq.hhh.retrofit2;
```

Comment:
Package name should be something meaningful like com.github.yassinelhadedy.retrofitrx